src/main/java/uk/gov/justice/digital/delius/transformers/InstitutionalReportTransformer.java

```diff
@@ -19,9 +22,34 @@ public InstitutionalReport institutionalReportOf(uk.gov.justice.digital.delius.j
             .institutionalReportId(report.getInstitutionalReportId())
             .offenderId(report.getOffenderId())
             .conviction(convictionOf(report))
+            .sentence(sentenceOf(report))
             .build();
     }
 
+    private Sentence sentenceOf(uk.gov.justice.digital.delius.jpa.standard.entity.InstitutionalReport report) {
+        return Optional.ofNullable(report.getCustody())
+            .filter(custody -> !convertToBoolean(custody.getSoftDeleted()))
```

Comment:
I know this does no harm but not sure in real world this could ever happen, e.g a Report associated with a deleted custody/disposal record